### PR TITLE
Update best_practices.rst

### PR DESCRIPTION
**Pull request by ifdattic**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.3 |
| Fixed tickets |  |

### CODE REVIEW

File: cookbook/bundles/best_practices.rst [hunk 1]
```diff
@@ -233,8 +233,8 @@ following standardized instructions in your ``README.md`` file.
     Step 2: Enable the Bundle
     -------------------------
 
-    Then, enable the bundle by adding the following line in the `app/AppKernel.php`
-    file of your project:
+    Then, enable the bundle by adding the following line in the
+    `app/AppKernel.php` file of your project:
```
Comment (xabbuh) [inline cookbook/bundles/best_practices.rst]: Actually, we also have to use double backticks here.

**Comment by xabbuh** [comment]
Hi Andrew, I really love all the great little fixes you do. But honestly, I'm beginning to loose track of what I have already reviewed and whether or not I have already seen a change in one of your other pull requests. That said, would you mind to keep one pull request power branch and update it when you find more things (of course only if they are roughly related)? I would really appreciate that. :)

**Comment by ifdattic** [comment]
I will try to keep than in mind, but I can't promise that :)

File: cookbook/bundles/best_practices.rst [hunk 2]
```diff
@@ -233,8 +233,8 @@ following standardized instructions in your ``README.md`` file.
     Step 2: Enable the Bundle
     -------------------------
 
-    Then, enable the bundle by adding the following line in the `app/AppKernel.php`
-    file of your project:
+    Then, enable the bundle by adding the following line in the
+    ``app/AppKernel.php`` file of your project:
```
Comment (xabbuh) [inline cookbook/bundles/best_practices.rst]: by the way, I don't think we should change the position of the line break here

Comment (WouterJ) [inline cookbook/bundles/best_practices.rst]: it should be single backticks, as it's markdown syntax

Comment (xabbuh) [inline cookbook/bundles/best_practices.rst]: Oh really, my bad. @WouterJ is right here. But then, I don't see why we should change anything here at all.

Comment (ifdattic) [inline cookbook/bundles/best_practices.rst]: It was to remove the horizontal scrollbar on smaller screens.

**Comment by WouterJ** [comment]
Put on hold for #4858

**Pull request closed without merging**